docs/source/system-architecture.md

```diff
@@ -291,6 +299,18 @@ located.
 
 See [Enabling IBM Spectrum Conductor Support](kernel-conductor.html#enabling-ibm-spectrum-conductor-support) for details.
 
+###### CustomResourceProcessProxy
+Enterprise Gateway also provides a implementation of a process proxy derived from KubernetesProcessProxy 
+called CustomResourceProcessProxy. 
+
+Instead of creating kernel based on Kubernetes pod, CustomResourceProcessProxy
+manages kernel by implementation of custom resource definition(CRD), e.g. SparkApplication is a CRD which include
+plenty component of a Spark on Kubernetes application.
```

Comment:
```suggestion
Instead of creating kernels based on a Kubernetes pod, `CustomResourceProcessProxy`
manages kernels via a custom resource definition (CRD). For example,  `SparkApplication` is a CRD that includes
many components of a Spark-on-Kubernetes application.
```

Reply:
ok, updated